react-challenge.html

```diff
@@ -67,28 +68,27 @@ <h1>Welcome to the Mood Meter App</h1>
       class MoodMeter extends React.Component{
         render(){
           return(
+              <div>
               <h3>Mood Options:</h3>
               <p style={{fontSize: "40px"}}>{ this.props.moodOptions }</p>
-              <button onClick={ }>Click Here For Next Mood Option</button>
+              <button onClick= {this.props.next} >Click Here For Next Mood Option</button>
               <br />
               <br />
               <h3>Save your mood here!</h3>
               <button onClick={ this.props.setThisMood }>Yup, this is me right now.</button>
               <h3>My mood:</h3>
               <p style={{fontSize: "40px"}}>{ this.props.set }</p>
-          )
+            </div>
```

Comment:
✔️ awesome! remember, you can also use `React.Fragment` to avoid adding extra unnecessary nodes to your HTML :)